commit 0fe584342da141957c8642191b508ad39e9b19e6
Author: Matthew Garrett <mjg@redhat.com>
Date:   Thu Sep 16 13:39:31 2010 -0400

    usbserial: Enable USB autosuspend by default on qcserial
    
    Seems to work fine in my testing.
    
    Signed-off-by: Matthew Garrett <mjg@redhat.com>

diff --git a/drivers/usb/serial/qcserial.c b/drivers/usb/serial/qcserial.c
index cde67ca..2846ad8 100644
--- a/drivers/usb/serial/qcserial.c
+++ b/drivers/usb/serial/qcserial.c
@@ -118,6 +118,8 @@ static int qcprobe(struct usb_serial *serial, const struct usb_device_id *id)
 
 	spin_lock_init(&data->susp_lock);
 
+	usb_enable_autosuspend(serial->dev);
+
 	switch (nintf) {
 	case 1:
 		/* QDL mode */
